src/hdf5/atms_netcdf_hdf5_2ioda.py

```diff
@@ -257,45 +246,45 @@ def get_WMO_satellite_ID(filename):
     return WMO_sat_ID
 
 
-def get_string_dtg(obs_time_utc):
+def get_observation_time(obs_time_utc, write_string=False):
 
     year = obs_time_utc[:, :, 0].flatten()
     month = obs_time_utc[:, :, 1].flatten()
     day = obs_time_utc[:, :, 2].flatten()
     hour = obs_time_utc[:, :, 3].flatten()
     minute = obs_time_utc[:, :, 4].flatten()
+    second = 0
     dtg = []
     for i, yyyy in enumerate(year):
-        cdtg = ("%4i-%.2i-%.2iT%.2i:%.2i:00Z" % (yyyy, month[i], day[i], hour[i], minute[i]))
-        dtg.append(cdtg)
+        if not write_string:
```

Comment:
Is this not more confusing than doing:
```
if write_string:
else:
```
and flip the if and else?

Reply:
sure flipped and just created string time once